spark/benchmarks/CometAggregateBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,464 @@
+================================================================================================
```

Comment:
We don't have other bench results under `spark/benchmarks`, so we may not need to commit this file...

Reply:
These result files are very specific to the hardware they are run on, so I'm not sure it is worth checking these in.

Reply:
deleted